examples/graphbolt/download.py

```diff
@@ -0,0 +1,577 @@
+import argparse, hashlib, os, shutil, tarfile, yaml
+import subprocess
+import urllib.request as ur
+
+import dgl.graphbolt as gb
+import numpy as np
+from tqdm import tqdm
+
+GBFACTOR = 1 << 30
+
+
+def build_yaml_helper(path, dataset_size, in_memory=True):
+    """The stirng to build the yaml file. (Still need modification)"""
+
+    data = {
+        "dataset_name": os.path.basename(path),
+        "feature_data": [
+            {
+                "domain": "node",
+                "format": "numpy",
+                "in_memory": in_memory,
+                "name": "feat",
+                "path": "data/paper_feat.npy",
```

Comment:
Is the embedding dimension for all datasets same, 1024? (homo hetero all sizes?)

Reply:
Yes. the embedding dimension are all 1024